lms/templates/course_modes/choose.html

```diff
@@ -12,6 +12,257 @@
     ${_("Enroll In {course_name} | Choose Your Track").format(course_name=course_name)}
 </%block>
 
+<%block name="testcss">
+<style>
+.v2.register-choice {
```

Comment:
Is there a need to inline this? If not, I'd add it to the existing stylesheet used by this page.